Automattic/wp-calypso
# Add ectoplasm color scheme (Method B)

One part of #47036
Method B is defined in #47103.  There's a "Method A" copy of the PR here: #47122.

#### Changes proposed in this Pull Request

* Add ectoplasm color scheme

#### Testing instructions

* Visit /me/account
* Choose ectoplasm and save

To see the wp-admin color scheme, create an atomic site, then visit /wp-admin/profile.php, choose color scheme and save.

Calypso:
![2020-11-03_16-12](https://user-images.githubusercontent.com/937354/98046239-76a75880-1def-11eb-9c97-db919c52e2db.png)

WP-Admin:
![2020-11-03_16-14](https://user-images.githubusercontent.com/937354/98046375-ba01c700-1def-11eb-8d36-7fa6cfe33ebc.png)

#### Known Issues:

* As I'm limited to the colors on https://color-studio.blog/, I couldn't find a good match for the green.  Purple is not perfect either but is closer.
* In calypso, the masterbar is a darker purple than the sidebar to match the other calypso themes.  In wp-admin, they're the same.
* Calypso "wants" the main button color to be a -50 lightness, which is darker than the wp-admin one.
* The wp-admin pallette shows an orange, but I couldn't find it used anywhere while using the site.
![2020-11-03_16-17](https://user-images.githubusercontent.com/937354/98046583-1238c900-1df0-11eb-93fe-a19abd2efdbb.png)
* Hovering in the sidebar flips the text color from white to black for contrast reasons:
![2020-11-03_16-18](https://user-images.githubusercontent.com/937354/98046654-37c5d280-1df0-11eb-9edb-b18f5b7873f8.png)
It doesn't flip in wp-admin:
![2020-11-03_16-18_1](https://user-images.githubusercontent.com/937354/98046681-43b19480-1df0-11eb-8dfd-ba45cb4f71d4.png)
* We would have to port these changes back to wp-admin.  But how?
